### DISCUSSION TITLE
It's a query: Where do I find the COSMOS-AzTEC-1 Galaxy

### DISCUSSION THREAD

**Discussion by ffmendez** [Stellarium/stellarium#2148] [General]
Hello everyone, first of all excellent work, when looking for the Galaxy: COSMOS-AzTEC-1, I did not find it in the catalog. my question is how can I update, or incorporate this object to stellarium

**Comment by gzotti** [comment]
Please see chapter 8 of the User Guide.

**Comment by ffmendez** [comment]
Thanks a lot, ...!
